library/Stratosphere/Check.hs

```diff
@@ -0,0 +1,34 @@
+module Stratosphere.Check
+     ( duplicateProperties
+     ) where
+
+-- Stratosphere.Check exports functions to catch errors
```

Comment:
Can you please make this a Haddock comment by using `-- |` instead of just `--`? That will make it show up in the Hackage docs.